index.js

```diff
@@ -1,24 +1,127 @@
-import express from 'express'
-import { ApolloServer } from 'apollo-server-express'
-import { resolvers } from './src/resolvers'
-import { typeDefs } from './src/typeDefs'
-import dotEnv from 'dotenv';
+const express = require('express');
+const JiraApi = require ('jira-client');
+require('dotenv').config();
 
-dotEnv.config()
-
-const PORT = process.env.PORT || 3500
-const app = express()
-
-const server = new ApolloServer({
-    typeDefs,
-    resolvers,
-    playground: true,
+const PORT = process.env.PORT || 9000;
+const app = express().use(express.urlencoded({extended: false})).use(express.json());
+const jira = new JiraApi({
+  protocol: 'https',
+  host: process.env.JIRA_HOST,
+  username: process.env.JIRA_USER,
+  password: process.env.JIRA_PASS,
+  apiVersion: '2',
+  strictSSL: false
 })
 
-server.applyMiddleware({ app })
+app.listen(PORT, () => { console.info(`Listening on Port# - ${PORT}`); })
 
 app.get('/', function(req, res) {
   res.send({ hello: 'there!' })
 })
 
-app.listen(PORT, () => `Listening at http://localhost:${PORT}`)
+app.post('/', async (req, res) => {
+  try {
+    if (req.body && req.body.type) {
+      switch(req.body.type) {
+        case 'ADDED_TO_SPACE':
+          return res.json({ 'text': `Thank you for adding me. Command \help` });
```

Comment:
You cant use /help to recieve the help message. I guess this should be just `help`

Reply:
Alright makes sense

Reply:
Added the change